chat/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "chat",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^1.4.0",
```

Comment:
I see you're not using this

Reply:
I tried to use it for the stream, but it was bad. removed :)